### PR TITLE
Update CHANGES

### PR DESCRIPTION
**Pull request by jonludlam**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -12,11 +12,32 @@ Tags:
...
+- Add a tooltip to references with text (@Julow, #945)
+- Add emoji to alerts in CSS (@yawaramin, #928)
 
 ### Fixed
 - Fix `--hidden` not always taken into account (@panglesd, #940)
+- Span labels in function arguments (@panglesd, #990)
+- Ensure generated html ends with a newline (@3Rafal, #954)
+- Warn against tags in pages (@Julow, #948) 
+- Remove unhelpful 'Unresolved_apply' errors (@gpetiot, #946)
+- Allow links and references in headings (@EmileTrotignon, #942)
```
Comment (panglesd) [inline CHANGES.md:31]: This was pair-programmed

```suggestion
- Allow links and references in headings (@EmileTrotignon, @panglesd, #942)
```

File: CHANGES.md [hunk 2]
```diff
@@ -12,11 +12,32 @@ Tags:
...
+- Fix rendering of method types (@zoggy, #935)
+- Fix section labelling with submodules (@EmileTrotignon, #931)
```
Comment (panglesd) [inline CHANGES.md:33]: This was pair-programmed

```suggestion
- Fix section labelling with submodules (@EmileTrotignon, @panglesd, #931)
```

File: CHANGES.md [hunk 3]
```diff
@@ -12,11 +12,32 @@ Tags:
...
-- Source code rendering (@Julow, @panglesd, #909)
+- Source code rendering (@Julow, @panglesd, @jonludlam #909, #996, #993, #982)
 - Handle tables markup (@panglesd, @gpetiot, #893)
+- Initial support for assets (@trefis, #975)
+- odoc-parser remerged (@jonludlam, #973)
+  This includes table support (@gpetiot, @panglesd, ocaml-doc/odoc-parser#11
+  ocaml-doc/odoc-parser#14) and delimited code blocks with optional output
+  (@jonludlam, ocaml-doc/odoc-parser#17)
+- Add a tooltip to references with text (@Julow, #945)
+- Add emoji to alerts in CSS (@yawaramin, #928)
```
Comment (panglesd) [inline CHANGES.md:24]:
```suggestion
- Add common language in shipped highlightjs (@Julow, #953)

```

File: CHANGES.md [hunk 4]
```diff
@@ -12,11 +12,32 @@ Tags:
...
 - Fix `--hidden` not always taken into account (@panglesd, #940)
+- Span labels in function arguments (@panglesd, #990)
+- Ensure generated html ends with a newline (@3Rafal, #954)
+- Warn against tags in pages (@Julow, #948) 
+- Remove unhelpful 'Unresolved_apply' errors (@gpetiot, #946)
+- Allow links and references in headings (@EmileTrotignon, #942)
+- Fix rendering of method types (@zoggy, #935)
+- Fix section labelling with submodules (@EmileTrotignon, #931)
+- LaTeX backend fixes (@Octachron, #921 #920)
+
```
Comment (panglesd) [inline CHANGES.md:35]:
```suggestion
- html: Remove extra space in class declarations (@Julow, #936)
- Fix rendering of unresolved references (@Julow, #957)
```

File: CHANGES.md [hunk 5]
```diff
@@ -12,11 +12,32 @@ Tags:
...
+- odoc-parser remerged (@jonludlam, #973)
+  This includes table support (@gpetiot, @panglesd, ocaml-doc/odoc-parser#11
+  ocaml-doc/odoc-parser#14) and delimited code blocks with optional output
+  (@jonludlam, ocaml-doc/odoc-parser#17)
+- Add a tooltip to references with text (@Julow, #945)
+- Add emoji to alerts in CSS (@yawaramin, #928)
 
 ### Fixed
 - Fix `--hidden` not always taken into account (@panglesd, #940)
+- Span labels in function arguments (@panglesd, #990)
```
Comment (panglesd) [inline CHANGES.md:27]: Maybe a more "user-oriented" description?
```suggestion
- Syntax highlight labels in function arguments (@panglesd, #990)
```

**Review by panglesd — APPROVED** [review]
Made some small comments!

File: CHANGES.md [hunk 6]
```diff
@@ -12,11 +12,32 @@ Tags:
 # 2.3.0 (Unreleased)
 
 ### Added
-- Source code rendering (@Julow, @panglesd, #909)
+- Source code rendering (@Julow, @panglesd, @jonludlam #909, #996, #993, #982)
 - Handle tables markup (@panglesd, @gpetiot, #893)
+- Initial support for assets (@trefis, #975)
+- odoc-parser remerged (@jonludlam, #973)
+  This includes table support (@gpetiot, @panglesd, ocaml-doc/odoc-parser#11
```
Comment (Julow) [inline CHANGES.md:19]: Things related to tables should be moved to "Handle tables markup" above.

The odoc-parser remerge should be in the "Changed" section (here it's in "Added").

Comment (jonludlam) [reply CHANGES.md:19]: Added seemed more appropriate to me, since we've added a whole opam package.

**Pull request merged**